test/lib/ansible_test/_data/sanity/validate-modules/validate_modules/schema.py

```diff
@@ -59,6 +60,12 @@ def collection_name(v, error_code=None):
     return v
 
 
+def deprecation_versions():
+    """Create a list of valid version for deprecation entries, current+4"""
+    major, minor = [int(version) for version in __version__.split('.')[0:2]]
+    return Any(*['{0}.{1}'.format(major, minor + increment) for increment in range(0, 5)])
```

Comment:
How about 
```python
from decimal import Decimal
```
and then
```suggestion
    ver = Decimal('.'.join(ansible.__version__.split('.')[:2]))
    return Any(*[str(ver + _ * Decimal('0.01')) for _ in range(5)])
```